client/components/ProductOverview/StyleSelector.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../../css/StyleSelector.scss';
+
+const StyleSelector = ({ styles, currentStyle, updateCurrentStyle }) => {
+  if (styles) {
```

Comment:
I would use a ternary operator here as well. After looking at this, seems like both ways are acceptable. https://reactjs.org/docs/conditional-rendering.html

Reply:
> Looks good so far. Are the styles the same as the thumbnails? I thought they were different for some reason?

Styles are essentially a specific set of thumbnails. The style selector lets you choose between the different sets by providing the first thumbnail from each set to click on.